### PR TITLE
Extend issue tracker interface to support actions (#504)

### PR DESCRIPTION
**Pull request by oliverchang**
Actions are modifications on an issue, e.g. a comment and/or modifying
labels.

### CODE REVIEW

File: src/python/issue_management/monorail/__init__.py [hunk 1]
```diff
@@ -103,15 +103,62 @@ def components(self):
     return self._monorail_issue.components
 
   @property
-  def comments(self):
+  def actions(self):
     """Get the issue comments."""
```
Comment (inferno-chromium) [inline src/python/issue_management/monorail/__init__.py]: nit: fix description

Comment (oliverchang) [reply src/python/issue_management/monorail/__init__.py]: done.

**Pull request merged**